<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert</class>
 <widget class="QMainWindow" name="insert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget
{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="update">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/del.png</normaloff>:/img/del.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>240</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="photograph">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/photograph.png</normaloff>:/img/photograph.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>240</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="gather">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>410</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>240</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="out">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>590</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/out.png</normaloff>:/img/out.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>147</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="num">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>345</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rsc.qrc">:/img/num_in.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>570</y>
      <width>345</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rsc.qrc">:/img/name_in.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="num_text">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>590</y>
      <width>180</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25px &quot;微软雅黑&quot;;
color: rgb(112, 112, 112);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入学号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_text">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>180</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25px &quot;微软雅黑&quot;;
color: rgb(112, 112, 112);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>410</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/no.png</normaloff>:/img/no.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>240</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/yes.png</normaloff>:/img/yes.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>240</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
